crates/dbsp/src/circuit/circuit_builder.rs

```diff
@@ -810,6 +810,10 @@ pub trait Node {
     /// another node).
     unsafe fn clock_end(&mut self, scope: Scope);
 
+    fn init_metrics(&mut self, _gid: &GlobalNodeId) {}
```

Comment:
I think this should be just `init()`.  The operator can perform any kind of initialization here.